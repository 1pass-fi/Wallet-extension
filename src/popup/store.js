import { composeWithDevTools } from 'redux-devtools-extension'
import { createStore, applyMiddleware } from 'redux'
import thunk from 'redux-thunk'

import rootReducer from './reducers/index'

const store = createStore(
  rootReducer,
  composeWithDevTools(applyMiddleware(thunk))
)

export default store
