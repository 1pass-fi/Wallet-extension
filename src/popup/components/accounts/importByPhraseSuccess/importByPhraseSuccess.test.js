import '@babel/polyfill'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { Provider } from 'react-redux'
import store from 'popup/store'
import React from 'react'

import ImportByPhraseSuccess from '.'

import { BrowserRouter as Router } from 'react-router-dom'

describe('Test for ImportByPhraseSuccess component', () => {
  describe('Render without crashing', () => {
    it('renders correctly', () => {
      const { container } = render(<Router><Provider store={store}><ImportByPhraseSuccess /></Provider></Router>)
      expect(container).toMatchSnapshot()
    })
  })
})
