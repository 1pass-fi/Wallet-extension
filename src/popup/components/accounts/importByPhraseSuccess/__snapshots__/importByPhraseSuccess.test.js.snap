// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for ImportByPhraseSuccess component Render without crashing renders correctly 1`] = `
<div>
  <div
    class="import-phrase-success"
  >
    <div
      class="congratulations-section"
    >
      <div
        class="title"
      >
        Congratulations!
      </div>
      <div
        class="details"
      >
        You successfully imported your wallet to Koi.
      </div>
    </div>
    <div
      class="tips-section"
    >
      <div
        class="title"
      >
        Remember to keep your seed phrase safe.
      </div>
      <div
        class="details"
      >
        <div
          class="safekeeping"
        >
          A few tips for safekeeping:
        </div>
        <div
          class="tips-set"
        >
          <p
            class="tip-item"
          >
            Save a backup copy in several places.
          </p>
          <p
            class="tip-item"
          >
            Never share your phrase with anyone. 
          </p>
          <p
            class="tip-item"
          >
            Stay safe from phishing scams— Koi will never ask you for your seed phrase.
          </p>
          <p
            class="tip-item"
          >
            If you have questions or see something suspicious, email us at 
            <a
              class="security-link"
            >
              security@openkoi.com
            </a>
            .
          </p>
        </div>
      </div>
    </div>
    <a
      href="/account"
    >
      <button
        class="button-shared button-filled back-button"
      >
        Back to Accounts
      </button>
    </a>
  </div>
</div>
`;
